setup.py

```diff
@@ -52,7 +50,9 @@
     'develop', 'release', 'bdist_egg', 'bdist_rpm',
     'bdist_wininst', 'install_egg_info', 'build_sphinx',
     'egg_info', 'easy_install', 'upload', 'bdist_wheel',
-    '--single-version-externally-managed',
+    '--single-version-externally-managed', 'install', 'build', 'build_ext',
```

Comment:
Historically, the reason that this code was there was specifically so that setuptools was not imported unless a command that needed it was issued. Setuptools is very invasive, and we wanted to avoid using it as much as possible. In particular the way that it installed everything as eggs was quite a problem. That latter problem is now solved in recent versions of setuptools. However, I hesitate in making the change that you suggest, as it makes setuptools a dependency, and I fear that I fail to see the benefit that it brings us.